assets/src/hooks/v2/use_api_response.tsx

```diff
@@ -124,23 +162,19 @@ const useApiResponse = ({
     try {
       const now = Date.now();
       const result = await fetch(apiPath);
-      const json = (await result.json()) as RawResponse;
+      const json = await result.json();
 
       if (json.force_reload) {
         window.location.reload();
       }
 
-      const apiResponse = rawResponseToApiResponse(json);
+      const apiResponse = responseHandler(json);
 
       if (apiResponse.state == "failure") {
-        doFailureBuffer(
-          lastSuccess,
-          failureModeElapsedMs,
-          setApiResponse,
-          apiResponse
-        );
+        doFailureBuffer(lastSuccess, setApiResponse, apiResponse);
       } else {
         setApiResponse((prevApiResponse) => {
+          // Prevent on simulation_success
           if (prevApiResponse != null && prevApiResponse.state !== "success") {
```

Comment:
If this does need updating, I would suggest defining an `isSuccess(response: ApiResponse)` helper function in case this check needs to happen somewhere else. (or actually, it's already needed [a few lines up from here](https://github.com/mbta/screens/blob/6f451ddb38fd1388b405944bd35e89dc9c9171a2/assets/src/hooks/v2/use_api_response.tsx#L94))

Reply:
Great catch. Definitely think we need to prevent these logs in simulations. They would just be redundant, or worst case they wouldn't be representative of actual issues on live screens. The helper function is a good idea so I got that added, but I wanted to run this by you one more time just for validation that I'm thinking it through correctly (it's Friday so my brain is mush). `isSuccess` as it is in my commit will for sure help us find successful responses. I think the problem is that `!isSuccess()` doesn't tell us on it's own if it was a failure. To combat this, I added a second part to just that conditional. Am I thinking about that correctly?